.travis.yml

```diff
@@ -3,7 +3,12 @@ go:
   - 1.5
```

Comment:
While you are here, can you make these all `.x` (`- 1.5.x`) so that the highest patch version is tested.